test_itebd  fails randomly

Here are the error message

```
=========================== short test summary info ============================
FAILED pytests/example/iTEBD/iTEBD_test.py::test_itebd - AssertionError: assert np.float64(1.7300214150139936e-08) < 1e-08
 +  where np.float64(1.7300214150139936e-08) = <ufunc 'absolute'>((-2.078545122699786 - -2.07854514))
 +    where <ufunc 'absolute'> = np.abs
=================== 1 failed, 6 passed, 1 warning in 12.25s ====================
```

See here for more details: https://github.com/Cytnx-dev/Cytnx/actions/runs/15527318150/job/43709352321

This test seems also failed randomly.

```
=========================== short test summary info ============================
FAILED pytests/example/iTEBD/iTEBD_tag_test.py::test_itebd - AssertionError: assert np.float64(7.287104741848793e-08) < 1e-08
 +  where np.float64(7.287104741848793e-08) = <ufunc 'absolute'>((-2.0785450671289527 - -2.07854514))
 +    where <ufunc 'absolute'> = np.abs
=================== 1 failed, 6 passed, 1 warning in 12.16s ====================
```

--- a/example/iTEBD/iTEBD.py
+++ b/example/iTEBD/iTEBD.py
@@ -44,8 +44,8 @@ def itebd_tfim(chi = 20, J = 1.0, Hx = 1.0, dt = 0.1, CvgCrit = 1.0e-10):
     #
     A = cytnx.UniTensor([cytnx.Bond(chi),cytnx.Bond(2),cytnx.Bond(chi)],labels=['a','0','b']);
     B = cytnx.UniTensor(A.bonds(),rowrank=1,labels=['c','1','d']);
-    cytnx.random.normal_(B.get_block_(),0,0.2);
-    cytnx.random.normal_(A.get_block_(),0,0.2);
+    cytnx.random.normal_(B.get_block_(), mean=0, std=0.2, seed=0);
+    cytnx.random.normal_(A.get_block_(),mean=0, std=0.2, seed=0);
     A.print_diagram()
     B.print_diagram()
     #print(A)

--- a/example/iTEBD/iTEBD_U1.py
+++ b/example/iTEBD/iTEBD_U1.py
@@ -47,9 +47,9 @@ def itebd_heisenberg(chi = 32, J  = 1.0, dt = 0.1, CvgCrit = 1.0e-12):
     B = cytnx.UniTensor([bd_mid,bdi,bdo],labels=['c','1','d']);
 
     for b in range(len(B.get_blocks_())):
-        cytnx.random.normal_(B.get_block_(b),0,0.2);
+        cytnx.random.normal_(B.get_block_(b), mean=0, std=0.2, seed=0);
     for a in range(len(A.get_blocks_())):
-        cytnx.random.normal_(A.get_block_(a),0,0.2);
+        cytnx.random.normal_(A.get_block_(a), mean=0, std=0.2, seed=0);
 
     A.print_diagram()
     B.print_diagram()

--- a/example/iTEBD/iTEBD_tag.py
+++ b/example/iTEBD/iTEBD_tag.py
@@ -46,8 +46,8 @@ def itebd_tfim_tag(chi = 20, J = 1.0, Hx = 1.0, dt = 0.1, CvgCrit = 1.0e-10):
                         cytnx.Bond(chi,BD_OUT)],labels=['a','0','b']);
     B = cytnx.UniTensor(A.bonds(),rowrank=1,labels=['c','1','d']);
 
-    cytnx.random.normal_(B.get_block_(),0,0.2);
-    cytnx.random.normal_(A.get_block_(),0,0.2);
+    cytnx.random.normal_(B.get_block_(), mean=0, std=0.2, seed=0);
+    cytnx.random.normal_(A.get_block_(), mean=0, std=0.2, seed=0);
     A.print_diagram()
     B.print_diagram()
     #print(A)

